### PR TITLE
Fix Internal Test

### PR DESCRIPTION
**Pull request by YingboMa**

### CODE REVIEW

File: test/internal.jl [hunk 1]
```diff
@@ -2,14 +2,13 @@ using SciCompDSL
 using Base.Test
 
 # `Expr`, `Number` -> `Operation`
-@test isequal(Operation(2), 2)
-expr = :(-inv(2sqrt(+(a, b))))
+@IVar a b x y
```
Comment (ChrisRackauckas) [inline test/internal.jl]: Does it only work for independent variables?

Comment (YingboMa) [reply test/internal.jl]: It works for all kinds of variables.

Comment (ChrisRackauckas) [reply test/internal.jl]: Should test that then. Does it still work when they have values?

File: test/internal.jl [hunk 2]
```diff
@@ -2,14 +2,15 @@ using SciCompDSL
 using Base.Test
 
 # `Expr`, `Number` -> `Operation`
-@test isequal(Operation(2), 2)
-expr = :(-inv(2sqrt(+(a, b))))
+@IVar a b
```
Comment (ChrisRackauckas) [inline test/internal.jl]: Parameters?

**Pull request merged**